### ISSUE TITLE
Finish writing test_optionalAmpListOmitted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4409]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4409|
|Type|task|
|Created|2010-04-25 23:23:58Z|
|Branch|https://github.com/twisted/twisted/tree/optional-amp-list-test-4409|

`twisted/test/test_amp.py` has a test, `test_optionalAmpListOmitted`, with no real content.

It should be written.

<details><summary>Searchable metadata</summary>

```
trac-id__4409 4409
type__task task
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_optional_amp_list_test_4409 branches/optional-amp-list-test-4409
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__amp__tests amp, tests
time__1272237838000000 1272237838000000
changetime__1273154845000000 1273154845000000
version__None None
owner__ 
cc__rwall
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [28870]) Branching to 'optional-amp-list-test-4409'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [28871]) Test a command with an optional AmpList with no supplied value

refs [#4409](https://github.com/twisted/twisted/issues/4409)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set owner to @exarkun|
|-|-|

 * New test function body makes sense
 * test docstring is clearer
 * All AMP tests pass

Please merge

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [28903]) Merge optional-amp-list-test-4409

Author: exarkun
Reviewer: rwall
Fixes: [#4409](https://github.com/twisted/twisted/issues/4409)

Finish writing an AMP test for the omission of a value for an
optional AmpList parameter.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
